Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -879,13 +880,24 @@ void MainWindow::OnImportNANDBackup()
   auto beginning = QDateTime::currentDateTime().toMSecsSinceEpoch();
 
   auto result = std::async(std::launch::async, [&] {
-    DiscIO::NANDImporter().ImportNANDBin(file.toStdString(), [&dialog, beginning] {
-      QueueOnObject(dialog, [&dialog, beginning] {
-        dialog->setLabelText(
-            tr("Importing NAND backup\n Time elapsed: %1s")
-                .arg((QDateTime::currentDateTime().toMSecsSinceEpoch() - beginning) / 1000));
-      });
-    });
+    DiscIO::NANDImporter().ImportNANDBin(
+        file.toStdString(),
+        [&dialog, beginning] {
+          QueueOnObject(dialog, [&dialog, beginning] {
+            dialog->setLabelText(
+                tr("Importing NAND backup\n Time elapsed: %1s")
+                    .arg((QDateTime::currentDateTime().toMSecsSinceEpoch() - beginning) / 1000));
+          });
+        },
+        [this] {
+          return RunOnObject(this, [this] {
+            return QFileDialog::getOpenFileName(this, tr("Select the OTP/SEEPROM dump"),
+                                                QDir::currentPath(),
+                                                tr("BootMii OTP/SEEPROM dump (*.bin);;"
```

Comment:
This won't let you reuse the wx translation, right? At least not unless you split the strings up and join them after translation.
Also: Does QT not require a file pattern as each even argument (complementing every odd argument which describes the string)? Either way that double semicolon looks odd to a non-QT guy like me.

Reply:
> This won't let you reuse the wx translation, right? At least not unless you split the strings up and join them after translation.

That's correct. We're handling other file pickers the same way, so this problem shouldn't block this PR.

Reply:
Just checked, the filter syntax for Qt is:

    Format (*.ext1 *.ext2 ...);;Another format (*.ext)

So I think the usage is correct here.

Reply:
Alright, fine by me then. The same probably goes for WX down there, so don't mind those comments.